tests/test_packages/test_skills/test_echo.py

```diff
@@ -76,6 +76,11 @@ def test_echo(self, pytestconfig):
         assert sent_envelope.protocol_id == received_envelope.protocol_id
         assert sent_envelope.message == received_envelope.message
 
-        self.terminate_agents([process])
+        check_string = "hello"
```

Comment:
Ok I can see this work for a single short string. How about a list of strings as mentioned in the issue?